model/subnet.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+// Subnet represents a parent subnet range.
+type Subnet struct {
+	ID             string
+	CIDR           string
+	AccountID      string
+	VPCID          string
+	ParentSubnet   string
```

Comment:
Maybe it would be worth renaming it to `ParentSubnetID` to be inline with other fields.

Reply:
+1

Reply:
Done thanks :)